docker/Dockerfile.php71

```diff
@@ -37,7 +37,7 @@ RUN debMultiarch="$(dpkg-architecture --query DEB_BUILD_MULTIARCH)"; \
 RUN pecl install APCu; \
     pecl install memcached; \
     pecl install redis; \
-    pecl install xdebug; \
+    pecl install xdebug-3.1.6; \
```

Comment:
that will not work, 3.1.6 claims it needs 7.2 or newer
![image](https://user-images.githubusercontent.com/2425577/214247657-41c3dc50-3ca5-45df-99a1-16e8b2dee0b3.png)
https://pecl.php.net/package/xdebug/3.1.6

Reply:
according to this page https://xdebug.org/download/historical 2.9.8 is the last one that mentions PHP 7.1

Reply:
I've fixed that in the last commit

Reply:
Thanks a lot 🎉